mapping-files/npm.json

```diff
@@ -0,0 +1,37 @@
+{
+  "react": "https://github.com/facebook/react",
+  "lodash": "https://github.com/lodash/lodash",
+  "prop-types": "https://github.com/facebook/prop-types",
+  "moment": "https://github.com/moment/moment",
+  "react-dom": "https://github.com/facebook/react",
+  "react-redux": "https://github.com/reduxjs/react-redux",
+  "components": "https://github.com/upstage/components",
+  "express": "https://github.com/expressjs/express",
+  "htm": "https://github.com/developit/htm",
+  "shared": "https://github.com/hutkev/shared",
+  "app": "https://www.npmjs.com/package/app",
+  "typeorm": "https://github.com/typeorm/typeorm",
+  "axios": "https://github.com/axios/axios",
+  "rxjs": "https://github.com/reactivex/rxjs",
+  "styled-components": "https://github.com/styled-components/styled-components",
+  "redux": "https://github.com/reduxjs/redux",
+  "puppeteer": "https://github.com/GoogleChrome/puppeteer",
+  "recompose": "https://github.com/acdlite/recompose",
+  "react-native": "https://github.com/facebook/react-native",
+  "src": "https://github.com/wlaurance/src",
```

Comment:
This looks odd, at least the weekly downloads are tiny compared to the others https://www.npmjs.com/package/src. Maybe one particular user was influencing this result. However, nothing to worry about.